### PR TITLE
replaced custom button with the html button

### PR DESCRIPTION
**Pull request by emmacode**

### CODE REVIEW

File: packages/frontend-app/src/app/components/Select/Select.scss [hunk 1]
```diff
@@ -11,6 +11,10 @@
     @apply w-full;
   }
 
+  &.button-transparent {
+    @apply outline-none font-light text-secondary50;
+  }
+
```
Comment (gbenga504) [inline packages/frontend-app/src/app/components/Select/Select.scss:17]: You don't need an extra classname, can you add this `@apply outline-none font-light text-secondary50;` directly to the select class. It is more intuitive

File: packages/frontend-app/src/app/components/Select/Select.tsx [hunk 2]
```diff
@@ -184,10 +184,8 @@ const Select: React.FC<IProps> = ({
         </div>
       )}
       <div className="relative" ref={dropdownSelectableAreaRef}>
-        <Button
-          variant="transparent"
-          htmlType="button"
-          className={classNames("select", {
+        <button
+          className={classNames("select button-transparent", {
```
Comment (gbenga504) [inline packages/frontend-app/src/app/components/Select/Select.tsx:188]: You don't need an extra classname i.e button-transparent. Apply your tailwind classes on the `select` classname

**Review by gbenga504 — CHANGES_REQUESTED** [review]
Very nice work, a few comment

File: packages/frontend-app/src/app/components/Select/Select.tsx (hunk 2 above)
Comment (emmacode) [reply packages/frontend-app/src/app/components/Select/Select.tsx:188]: noted

**Review by gbenga504 — APPROVED** [review]
LGTM!

**Pull request merged**